From 218e9e3a2ee2bddf8728e5841b3c7fd8acf10778 Mon Sep 17 00:00:00 2001
From: Alexandru Costache <alexandru@balena.io>
Date: Tue, 12 May 2020 10:39:58 +0200
Subject: [PATCH] imx8mm-var-dart-nrt: Enable M4, run from TCM

Enable the M4 core, set firmware binary name,
set the DTB and let the binary reside in the TCM
(as it came by default).

Also, we stay on default DTB for now, as the M4 dtb:
- disables ecspi1 (no spidev)
- reserves DDR memory for the M4
- sets up rpmsg for M4

Upstream-status: Inappropriate [configuration]
Signed-off-by: Alexandru Costache <alexandru@balena.io>
---
 include/configs/imx8mm_var_dart.h | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/include/configs/imx8mm_var_dart.h b/include/configs/imx8mm_var_dart.h
index 86424e4..0edcc3c 100644
--- a/include/configs/imx8mm_var_dart.h
+++ b/include/configs/imx8mm_var_dart.h
@@ -115,9 +115,9 @@
 	"mmcautodetect=yes\0" \
 	"mmcpart=1\0" \
 	"m4_addr=0x7e0000\0" \
-	"m4_bin=hello_world.bin\0" \
-	"use_m4=no\0" \
-	"loadm4bin=load mmc ${mmcdev}:${mmcpart} ${loadaddr} ${bootdir}/${m4_bin}; " \
+	"m4_bin=m4_fw.bin\0" \
+	"use_m4=yes\0" \
+	"loadm4bin=load mmc ${mmcdev}:${resin_boot_part} ${loadaddr} ${m4_bin}; " \
 		"cp.b ${loadaddr} ${m4_addr} ${filesize}\0" \
 	"runm4bin=" \
 		"if test ${m4_addr} = 0x7e0000; then " \
-- 
2.7.4

